test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FileHash.Tests.ps1

```diff
@@ -77,4 +77,10 @@ Describe "Get-FileHash" -Tags "CI" {
             $result.Path | Should -Be $testDocument
         }
     }
+    
+    Context "Close file before writing hash details to pipeline" {
+        It "Should not hold the file open unnecessarily" {
+            {Get-FileHash $testDocument | Rename-Item -NewName {$_.Hash}} | Should -Not -Throw
```

Comment:
Does Rename-Item throw a terminating error if it can't get at the file? Wondering if we need `-ErrorAction Stop` on this one.

We should probably add a comment here indicating that this changes the filename of `$testDocument` in case folks try to add further tests after this point using the same file. Alternatively, you could store the new file name and rename it back in an `AfterAll` block?

Reply:
Good points; I've added `-ErrorAction Stop`, and updated the test to make a copy of the document first, and remove it afterwards.